tests/passes/static-par-conv/diff-num-cycles.expect

```diff
@@ -0,0 +1,35 @@
+component main(@go go: 1, @clk clk: 1, @reset reset: 1) -> (@done done: 1) {
+  cells {
+  }
+  wires {
+    group A0 {
+    }
+    group A1 {
+    }
+    group A2 {
+    }
+    group B0 {
+    }
+    group B1 {
+    }
+    group B2 {
+    }
+  }
+
+  control {
+    @static(7) seq {
+      @static(2) par {
+        @static(2) A0;
+        @static(2) B0;
+      }
+      @static(4) par {
+        @static(3) A1;
+        @static(4) B1;
```

Comment:
Why is it okay to merge `seq` that take different lengths of time? Is this using the "slack" concept we discussed?

Reply:
Yeah, similar idea. As long as the control statement with more cycles is coming from the seq w/ more statements overall, then we're good. Say the longest original seq is 
`seq {A; B ; C; D; E; F; G}`
Then we know we'll need 7 par blocks. And the maximum number of cycles for the first par block is going to be A, max for the second par block is going to be B, etc.